### PR TITLE
add soares2024fair

### PR DESCRIPTION
**Pull request by pedropaulofb**
add all necessary files for new dataset.

### CODE REVIEW

**Review by tgoprince — CHANGES_REQUESTED** [review]

File: models/soares2024fair/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+title: Conceptual Model of a FAIR Metadata Schema
+acronym: FAIR Metadata Schema
+issued: 2024
+modified:
+contributor:
+ - 0000-0002-0674-7960
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/soares2024fair/metadata.yaml:6]: Done. I changed the ORCIDs for DBLP IDs, as it is the catalog's recommended option.

File: models/soares2024fair/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - data publication
+ - interoperability
+context:
+ - research
+source:
+ - https://www.alice.cnptia.embrapa.br/alice/handle/doc/1169610
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/soares2024fair/metadata.yaml:41]: Done.

File: models/soares2024fair/references.bib [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  year      = {2024},
+  booktitle = {ER Forum, Special Topics, Posters and Demos: Companion Proceedings of the 43rd International Conference on Conceptual Modeling (ER 2024), Pittsburgh},
+  organization = {International Conference on Conceptual Modeling},
+  series    = {CEUR Workshop Proceedings},
+  volume    = {3849},
+  issn      = {1613-0073},
+  url       = {https://www.alice.cnptia.embrapa.br/alice/bitstream/doc/1169610/1/AA-Towards-conceptual-2024.pdf},
+  note      = {Published in ER-Companion 2024, edited by Enrico Gallinucci, Hasan Yasar, Peter Chen, Sotirios Liaskos, Patrick Marcel, Sergio de Cesare, and Frederik Gailly},
+  keywords  = {Metadata schema, FAIR data principles, Conceptual modeling, Semantic web},
+  access    = {openAccess}
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/soares2024fair/references.bib:13]: Done.

File: models/soares2024fair/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+ - fair principles
+ - data schema
+theme: Class T - Technology
+editorialNote: 
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - data publication
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/soares2024fair/metadata.yaml:36]: I removed 'data publication'. However, I believe 'interoperability' still holds based on the following: "The proposed conceptual model and the FAIR assessment methodology help communities identify and address gaps in their metadata schemas to achieve full FAIR compliance.". Please let me know if you disagree so I can remove it.

File: models/soares2024fair/references.bib (hunk 3 above)
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/soares2024fair/references.bib]: All necessary modifications were done. You can please proceed with the review.

**Review by tgoprince — APPROVED** [review]

**Pull request merged**